search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/FormFilterTransformer.java

```diff
@@ -37,15 +40,11 @@ public List<String> toIndices(final FormFilter filter) {
     return Arrays.asList("tasklist-form-8.4.0_");
   }
 
-  private SearchQuery getFormByKeysQuery(final List<Long> formKey) {
-    final List<String> formKeyAsString = formKey.stream().map(String::valueOf).toList();
-
-    return stringTerms("id", formKeyAsString);
+  private SearchQuery getFormByKeysQuery(final List<Long> formKeys) {
```

Comment:
:magic_wand: Maybe for beautification reasons rename this to _getFormsByKeyQuery_ same as the next on _getFormsByIdQuery_ or if they are only used for singular results make them separate for singular and collections?

Reply:
Promise I will do this in a future PR 👍